app/scripts/services/BlocChatCookies.js

```diff
@@ -1,18 +1,28 @@
 (function() {
-  function BlocChatCookies($cookies, $uibModal) {
-    
-    var currentUser = $cookies.get('blocChatCurrentUser');
-    if (!currentUser || currentUser === '') {
+  function BlocChatCookies($cookies, $cookieStore, $uibModal) {
+
+    this.currentUser = $cookies.get('blocChatCurrentUser');
+
+    if (!this.currentUser || this.currentUser === '') {
       $uibModal.open({
         templateUrl: '/templates/usernameprompt.html',
         controller: 'UsernamePromptCtrl as user',
         backdrop: 'static',
         keyboard: false
       });
     }
+
+    this.getCurrentUser = function() {
```

Comment:
I like the idea of having a getter.. although its not necessary since `this.currentUser` is also accessible to anything consuming this service.